modules/nickserv.cpp

```diff
@@ -119,9 +141,18 @@ class CNickServ : public CModule {
             MCString msValues;
             msValues["password"] = GetNV("Password");
             PutIRC(CString::NamedFormat(GetNV("IdentifyCmd"), msValues));
+        } else if(Nick.NickEquals(sNickServName) && sMessage.find("Password accepted") != CString::npos && m_bJoinAfterIdentified) {
```

Comment:
@DarthGandalf 
yes, you're right. but is there any other way to do it?
one way would be that the user has to set keywords for example like "Password accepted", but for each network specifically
or is there a better alternative?